plugin/pkg/scheduler/factory/factory.go

```diff
@@ -191,6 +192,33 @@ func NewConfigFactory(client clientset.Interface, schedulerName string, hardPodA
 	return c
 }
 
+// GetNodeStore provides the cache to the nodes, mostly internal use, but may also be called by mock-tests.
+func (c *ConfigFactory) GetNodeStore() cache.Store {
+	return c.nodeLister.Store
+}
+
+func (c *ConfigFactory) GetHardPodAffinitySymmetricWeight() int {
+	return c.hardPodAffinitySymmetricWeight
+}
+
+func (c *ConfigFactory) GetFailureDomains() []string {
+	return c.failureDomains
+}
+
+func (f *ConfigFactory) GetSchedulerName() string {
+	return f.schedulerName
+}
+
+// GetClient provides a kubernetes client, mostly internal use, but may also be called by mock-tests.
+func (f *ConfigFactory) GetClient() clientset.Interface {
+	return f.client
+}
+
+// GetScheduledPodListerIndexer is another (self-explanatory) function only used in rare external circumstances (i.e. testing)
```

Comment:
For consistency with other comments (thanks for changing them):
// GetScheduledPodListerIndexer provides a pod lister, mostly internal use, but may also be called by mock-tests.